charts/azuremonitor-containers/templates/ama-logs-daemonset.yaml

```diff
@@ -46,6 +47,56 @@ spec:
        - name: TOKEN_NAMESPACE
          value: "azure-arc"
 {{-  .Values.Azure.Identity.MSIAdapterYaml | nindent 7 }}
+     {{- else }}
+     - name: msi-adapter
+       env:
+       - name: TOKEN_NAMESPACE
+         value: azure-arc
```

Comment:
include this environment variable also

       - name: AZMON_COLLECT_ENV
         value: "false"

Reply:
added